build.gradle

```diff
@@ -16,45 +16,52 @@
  */
 
 plugins {
-    id 'net.wooga.plugins' version '1.4.0'
+    id 'net.wooga.plugins' version '3.0.0-rc.2'
 }
 
 group 'net.wooga.gradle'
 description = 'Node release plugin for Gradle.'
 
 repositories {
-    jcenter()
     mavenCentral()
     maven {
         url "https://plugins.gradle.org/m2/"
     }
 }
 
 dependencies {
-    testCompile('org.jfrog.artifactory.client:artifactory-java-client-services:2.+') {
+    testImplementation('org.jfrog.artifactory.client:artifactory-java-client-services:2.9.2') {
         exclude module: 'logback-classic'
     }
-    testCompile 'org.kohsuke:github-api:latest.release'
-    testCompile 'com.github.stefanbirkner:system-rules:1.18.0'
+    testImplementation 'org.kohsuke:github-api:1.301'
+    testImplementation 'com.wooga.gradle:gradle-commons-test:0.3.0'
 
-    compile 'com.netflix.nebula:nebula-release-plugin:6.+'
-    compile 'com.moowork.gradle:gradle-node-plugin:1.2.0'
-    compile "gradle.plugin.net.wooga.gradle:atlas-github:1.+"
-    compile "gradle.plugin.net.wooga.gradle:atlas-release:1.+"
-    compile gradleApi()
-    compile localGroovy()
+    // https://plugins.gradle.org/plugin/org.ajoberstar.grgit
+    implementation 'org.ajoberstar.grgit:grgit-core:(4.1,5]'
+    implementation 'org.ajoberstar.grgit:grgit-gradle:(4.1,5]'
+    // https://github.com/node-gradle/gradle-node-plugin
+    implementation "com.github.node-gradle:gradle-node-plugin:3.2.1"
+
+    // Internal wooga plugins
+    implementation 'com.wooga.gradle:gradle-commons:0.4.1'
+    implementation 'gradle.plugin.net.wooga.gradle:atlas-version:1+'
+    implementation 'gradle.plugin.net.wooga.gradle:atlas-github:(2, 3]'
+    implementation 'gradle.plugin.net.wooga.gradle:atlas-release:(2, 3]'
```

Comment:
Is this really stil needed?

Reply:
I will check.

Reply:
Yes.

Reply:
What is pulled from the release plugin?

Reply:
It was pulling the predicate being used for the release task but now a custom one which you wrote is the one used.